packages/backend/src/configuration/environment.ts

```diff
@@ -34,47 +40,48 @@ const envVarsSchema = object({
   MOODLE_BASE_URL: string()
     .required()
     .uri()
-    .description('URL of your Moodle instance, eg. https://moodle.domain.me'),
+    .description('URL of your Moodle instance, eg. https://moodle.domain.me .'),
```

Comment:
```suggestion
    .description('URL of your Moodle instance, eg. https://moodle.example.com .'),
```
Is this supposed to be advertising otherwise I would use the `example.com` domain.